### DISCUSSION TITLE
How do I charge gas for transactions

### DISCUSSION THREAD

**Discussion by antonydenyer** [ConsenSys/quorum#1291] [Q&A] [answered]
How do I charge gas for transactions in a private GoQuorum network

**Comment by baptiste-b-pegasys** [comment, marked as answer]
Gas is not enabled in GoQuorum. We'd suggest using Besu if you want to use gas in your network.
https://besu.hyperledger.org/en/stable/ ; https://github.com/hyperledger/besu

There might be some plan to enable gas in future, but many questions are rised on security of private transactions
